android/gradle.properties

```diff
@@ -1 +1,8 @@
 android.useAndroidX=true
+BitmovinPlayerReactNative_kotlinVersion=1.7.0
+BitmovinPlayerReactNative_minSdkVersion=21
+BitmovinPlayerReactNative_targetSdkVersion=31
+BitmovinPlayerReactNative_compileSdkVersion=31
```

Comment:
Did we change this from 33?

Reply:
Yes, good eye I didn't notice this change. 
https://github.com/bitmovin/bitmovin-player-react-native/pull/318/commits/b1bb85e32a11a5e73996b56f6b3d06c1a02b9dbd